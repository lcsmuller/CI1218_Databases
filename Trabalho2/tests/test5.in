1  4  R X
2  5  R X
3  5  W X
4  4  W X
5  5  C -
6  4  C -