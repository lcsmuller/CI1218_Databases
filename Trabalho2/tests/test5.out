1 4,5 NS NV
